lib/grp.gi

```diff
@@ -4620,6 +4629,23 @@ InstallMethod( MaximalNormalSubgroups, "for simple groups",
               function(G) return [ TrivialSubgroup(G) ]; end);
 
 
+#############################################################################
+##
+#M  MaximalNormalSubgroups( <G> )
+##
+InstallMethod( MaximalNormalSubgroups, "general method selection",
+              [ IsGroup ],
+    function(G)
+
+    if 0 in AbelianInvariants(G) then
+      # (p) is a maximal normal subgroup in Z for every prime p
+      Error("number of maximal normal subgroups is infinity");
```

Comment:
@fingolfin Previously, when it returned fail, I had a test for it, that is not a problem. 

But I have no idea how to write a test for an error into a tst file. Should I just copy everything from my GAP output to the tst file? Or does it have a special method? Is the output even stable (apart from the error message)?